src/app/containers/ScheduleSummary.tsx

```diff
@@ -54,38 +57,47 @@ class ConnectedScheduleSummary extends React.Component<ConnectedScheduleSummaryP
                 return 'wednesday';
             case 4:
                 return 'thursday';
-            case 5: 
+            case 5:
                 return 'friday';
-            case 6: 
+            case 6:
                 return 'saturday';
-            case 7: 
+            case 7:
                 return 'sunday';
             default:
                 return '';
         }
     }
-    
+
     createWeekStatus() {
-        const { 
-            leaves, 
-            shifts, 
-            visibleTimeStart, 
+        const {
+            leaves,
+            shifts,
+            visibleTimeStart,
             visibleTimeEnd,
+            // tslint:disable-next-line:no-shadowed-variable
```

Comment:
I think we should try to respect this tslint issue.  Shadowing variable names can make the code less maintainable since refactoring variables might inadvertently rename functions that have the same name